frontend/scripts/constants.js

```diff
@@ -1,3 +1,6 @@
+export const ACTORS_COLUMN_IDS = [6, 7];
```

Comment:
evil evil evil!
you are hardcoding things that can easily change, effectively tying the frontend to a specific database value.

Reply:
Totally agree. Given that this was an ID I supposed it was more fixed that the type prop. Will change to `type` property. 👍